test/hs30.jl

```diff
@@ -0,0 +1,24 @@
+# HS problem 30 in NLS format
+#
+#   Source:
+#   W. Hock and K. Schittkowski,
+#   Test examples for nonlinear programming codes,
+#   Lecture Notes in Economics and Mathematical Systems 187,
+#   Springer Verlag Berlin Heidelberg, 1981
+#   10.1007/978-3-642-48320-2
+#
+# A. S. Siqueira, Curitiba/BR, 04/2018.
+
+export hs30
+
+"Hock-Schittkowski problem 30 in NLS format"
+function hs30()
+
+  model = Model()
+  lvar = [1.0; -10.0; -10.0]
+  @variable(model, lvar[i] <= x[i=1:3] <= 10, start=1.0)
+  @NLexpression(model, F[i=1:3], x[i] + 0.0)
```

Comment:
Why do we need to add `+ 0.0` here?

Reply:
`x[i]` alone raises an error. It's treated differently by JuMP, but I did not investigate if it can be solved on our side.
`1 * x[i]` below also works, apparently.